src/theme/utils.ts

```diff
@@ -26,7 +26,7 @@ export const calcSize = (expr?: string) => (props: WrappedTheme) => {
 
 export const getColor = (colorPath: string[]) => getOrElse(["colors", ...colorPath], "#fff")
 
-export const getSizeBy = (multiplier: number) => (props: WrappedTheme) => {
+export const getSizeBy = (multiplier: number = 1) => (props: WrappedTheme) => {
   const size = (getSizeUnit(props) || 0) * multiplier
```

Comment:
0 is not the best fallback value for sizes. May be will be better to implement fallback in **getSizeUnit** function?
```typescript
  const size = (getSizeUnit(props) || 1) * multiplier
```